front-end/src/app/transactions/transaction-group-f/transaction-group-f.component.ts

```diff
@@ -38,8 +43,32 @@ export class TransactionGroupFComponent extends TransactionTypeBaseComponent imp
     protected override transactionService: TransactionService,
     protected override validateService: ValidateService,
     protected override fb: FormBuilder,
-    protected override router: Router
+    protected override router: Router,
+    protected activatedRoute: ActivatedRoute
   ) {
     super(messageService, transactionService, validateService, fb, router);
+
+    of(this.activatedRoute.snapshot.data['transactionType']).subscribe((transactionType: TransactionType) => {
```

Comment:
https://angular.io/api/router/ActivatedRoute#description shows how activatedRoute provides the observable for you to subscribe to.